engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -52,12 +55,27 @@ should avoid installing extra or unnecessary packages just because they
 might be “nice to have.” For example, you don’t need to include a text editor
 in a database image.
 
-### Run only one process per container
-
-In almost all cases, you should only run a single process in a single
-container. Decoupling applications into multiple containers makes it much
-easier to scale horizontally and reuse containers. If that service depends on
-another service, make use of [container linking](../../userguide/networking/default_network/dockerlinks.md).
+### Each container should have only one concern
+
+Decoupling applications into multiple containers makes it much easier to scale
+horizontally and reuse containers.  For instance, in a web application stack you
```

Comment:
"For instance, a web application stack might consist of three...."

Maybe you could use Mermaid to make a little graph. @johndmulhausen has the details.